Documentation/UPI/vSphere_terraform/README.md

```diff
@@ -0,0 +1,201 @@
+| master3.okd.example.com | 10.20.15.13 | 00:1c:14:00:00:13 |
+| etcd-0.okd.example.com | 10.20.15.11 | |
+| etcd-1.okd.example.com | 10.20.15.12 | |
+| etcd-2.okd.example.com | 10.20.15.13 | |
+| worker1.okd.example.com | 10.20.15.41 | 00:1c:14:00:00:41 |
+| worker2.okd.example.com | 10.20.15.42 | 00:1c:14:00:00:42 |
+| worker3.okd.example.com | 10.20.15.43 | 00:1c:14:00:00:43 |
+
+#### SRV-records
+
+| domain | TTL | priority | weight | port | target |
+|--------|-----|----------|--------|------|--------|
+| _etcd-server-ssl._tcp.okd.example.com | 86400 | 0 | 10 | 2380 (etcd) | 10.20.15.11, 10.20.15.12, 10.20.15.13
+
+#### wildcard DNS record
+
+| domain | A-record | comment |
+|--------|----------|---------|
+| *.apps.okd.example.com | 10.20.15.41, 10.20.15.42, 10.20.15.43 | For high availability, we choose the first three worker nodes, which should run the router component |
+
+#### Loadbalancer - HAProxy configuration
+
+Note: Binding the API port also to 443 allows us to connect easyly with `oc`, no need for port number, e.g. `oc login api.okd.example.com`
+
+```HAProxy
+global
+    log         127.0.0.1 local2
+
+    chroot      /var/lib/haproxy
+    pidfile     /var/run/haproxy.pid
+    maxconn     4000
+    user        haproxy
+    group       haproxy
+    daemon
+
+    stats socket /var/lib/haproxy/stats
+
+    ssl-default-bind-ciphers PROFILE=SYSTEM
+    ssl-default-server-ciphers PROFILE=SYSTEM
+
+defaults
+    mode                    http
+    log                     global
+    option                  httplog
+    option                  dontlognull
+    timeout connect         10s
+    timeout client          1m
+    timeout server          1m
+
+listen  stats
+    bind 10.20.15.2:9000
+    mode            http
+    log             global
+    stats enable
+    stats refresh 30s
+    stats show-node
+
+frontend openshift-api-server
+    bind *:6443
+    bind *:443
+    default_backend openshift-api-server
+    mode tcp
+    option tcplog
+
+backend openshift-api-server
+    balance source
+    mode tcp
+    server btstrap 10.20.15.3:6443 check
+    server master1 10.20.15.11:6443 check
+    server master2 10.20.15.12:6443 check
+    server master3 10.20.15.13:6443 check
+
+frontend machine-config-server
+    bind *:22623
+    default_backend machine-config-server
+    mode tcp
+    option tcplog
+
+backend machine-config-server
+    balance source
+    mode tcp
+    server btstrap 10.20.15.3:22623 check
+    server master1 10.20.15.11:22623 check
+    server master2 10.20.15.12:22623 check
+    server master3 10.20.15.13:22623 check
+```
+
+## Get openshift-install
+
+Download oc from  <https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/>
+
+Choose a release ([dev-releases](https://origin-release.svc.ci.openshift.org/), ...)
+
+Extract `openshift-install` tool (e.g. `oc adm release extract --tools registry.svc.ci.openshift.org/origin/release:4.4.0-0.okd-2020-01-10-182730`)
```

Comment:
Note that it would also extract `oc`. `oc adm release extract --command=openshift-install` would extract just the installer - I think that would be preferable here